pages/login-notice/[type].tsx

```diff
@@ -42,21 +31,28 @@ export const NOTICE_CONTENT = {
       ...getNotificationContent('manage your notifications'),
       'If you want to unsubscribe from notifications without creating a GOV.UK One Login, you can use the unsubscribe link in the emails we send to you.',
     ],
-    redirectUrl: notificationRoutes.manageNotifications,
+    redirectUrl: `${notificationRoutes.manageNotifications}?migrationType=${MANAGE_NOTIFICATIONS}`,
   }),
   [SUBSCRIPTION_NOTIFICATIONS]: (ctx) => ({
     title: 'Sign up for updates',
     content: getNotificationContent('sign up for updates'),
-    redirectUrl: `${notificationRoutes.manageNotifications}?action=${URL_ACTIONS.SUBSCRIBE}&grantId=${ctx.query.grantId}`,
+    redirectUrl: `${notificationRoutes.manageNotifications}?action=${URL_ACTIONS.SUBSCRIBE}&grantId=${ctx.query.grantId}&migrationType=${SUBSCRIPTION_NOTIFICATIONS}`,
   }),
   [SAVED_SEARCH]: (ctx) => ({
     title: 'Save your search',
     content: getNotificationContent('save a search'),
     redirectUrl: `${notificationRoutes.saveSearch}?${buildQueryString(
       ctx.query,
-    )}`,
+    )}&migrationType=${SAVED_SEARCH}`,
+  }),
+  [NEWSLETTER]: () => ({
+    title: 'Get updates about new grants',
+    content: [
+      'We will only ever email you once a week. We will not email you if no new grants have been added.',
+      'If you do not have a GOV.UK One Login, you can create one.',
+    ],
+    redirectUrl: `${notificationRoutes.manageNotifications}?action=${URL_ACTIONS.NEWSLETTER_SUBSCRIBE}&migrationType=${NEWSLETTER}`,
```

Comment:
This isn't actually used (newsletter gets its own page), but may as well add the correct content here in case we get round to this refactor